dailystats.py

```diff
@@ -1,12 +1,12 @@
+"""Identify how well the different English regions are responding to lockdown.
+
+Assumption is that the peak occurs on April 8th (2 weeks after lockdown began)
+Some areas are responding better than others. This reports a progress factor for each region which is computed by
+taking the average daily case count since the "peak" and dividing that into the daily case count on April 8th
+Logic is a little flawed but provides a useful indicator for identification of what is happening below England level
+"""

```

Comment:
This style of docstring has a functional effect - the documentation is actually attached to the module - e.g.
```
import dailystats

print(dailystats.__doc__)
```

Would print out the message. This is quite useful for learning about objects interactively. Same applies to functions and classes